core/controllers/feedback_test.py

```diff
@@ -515,11 +516,9 @@ def _accept_suggestion(self, thread_id, csrf_token,
     def _reject_suggestion(self, thread_id, csrf_token,
```

Comment:
If I'm not mistaken, what's being done below is to generate the full thread id from the exp id and thread id, then take just the thread id bit here. Wouldn't it be better to just use the thread id directly?

Ditto for accept_suggestion, above. In general I would suggest minimizing the use of full_thread_id where possible -- it becomes particularly confusing when it's called thread id.